src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -354,4 +354,7 @@
   <data name="net_http_unix_invalid_response" xml:space="preserve">
     <value>The server returned an invalid or unrecognized response.</value>
   </data>
-</root>
+  <data name="Argument_BufferNotLongEnough" xml:space="preserve">
+    <value>The buffer was not long enough.</value>
+  </data>
+</root>
```

Comment:
I agree that this was a change, but "it wasn't my fault" in that I used the editor in VS (so likely to show up in commits from others).  I can certainly put this back externally, but none of the other resource files I modified ended in a new line; should I conform those as well, or that will be handled via tool later?